subsys/usb/class/audio/usb_audio_internal.h

```diff
@@ -349,7 +384,7 @@ struct dev##_descriptor_##i {						\
 	struct usb_if_descriptor as_interface_alt_0;			\
 	struct usb_if_descriptor as_interface_alt_1;			\
 		struct as_cs_interface_descriptor as_cs_interface;	\
-		struct format_type_i_descriptor format;			\
+		struct dev##_format_type_i_descriptor format;		\
```

Comment:
The sample does not build without this change.
``` diff
-		struct dev##_format_type_i_descriptor format;		\
+		struct dev##_format_type_i_descriptor format_##i;	\
```

Reply:
Thanks, fixed it now.